### DISCUSSION TITLE
Check if Sonoff RF Bridge R2 hardware direct/bypass hack works

### DISCUSSION THREAD

**Discussion by mortenmoulder** [arendst/Tasmota#19464] [Support request]
I have successfully installed Tasmota on my new RF Bridge R2 2.2. I used RFLink32 before, which showed that messages are coming in, but they are not being decoded.

I cut the traces according to this documentation: https://github.com/schmurtzm/RFLink32-For-Sonoff-RF-Bridge

And then I applied this configuration to Tasmota: `{"NAME":"RFB v2.2","GPIO":[1,1,1,1,1152,1120,1,1,1,288,1,1,1,1],"FLAG":0,"BASE":18}`

Then I opened the console and pressed some buttons on my remote, but nothing showed up. I am not entirely sure what to do.

My remote is a Dooya DC90 for my awning.

**Comment by Baggerpk** [comment]
Hi.. I got the same Problem.. How can help?

**Reply by Baggerpk** [reply]
I found out the Problem  !
The Sonoff RF Bridge R2 has to be flashed with  tasmota-sensors.bin !

**Comment by vrublevskys** [comment]
Hi! Got the same issue. Does anybody understand whats going wrong?

**Reply by sfromis** [reply]
Did you try the already suggested solution?

**Reply by vrublevskys** [reply]
Yes, I flashed tasmota-sensors.bin

**Reply by ruun3r** [reply]
> Yes, I flashed tasmota-sensors.bin

I did that too, but the remote is not recognised. Other 433 devices show up.
What version of tasmota-sensors.bin did you use?

**Reply by sfromis** [reply]
As RF is a bad mess of non-standards, it should not even be surprising that not all senders are being picked up. This is no "issue", just badly outdated tech. Also, some devices use rolling codes, which is outside of what's supported.

**Reply by ruun3r** [reply]
I was hopeful as I read that it worked for @Baggerpk. So I wanted to know how he achieved it.

The two Dooya DC90 I own, fortunately have constant codes.

**Reply by sfromis** [reply]
Here's an old thread about trouble with Dooya DC90 and rc-switch. I did not read it through to arrive at a conclusion for what to expect or not. https://github.com/sui77/rc-switch/issues/54

**Reply by Baggerpk** [reply]
hi.. here i am again..
sorry that my english ist not so good. because i am a german.
I found out, that it only works with older version of SONOFF RF BRIDGE .. NOT R2 !
It works with v1 (black housing) and Tasmota 9.1.0
I did it at the same way of the video ( https://www.youtube.com/watch?v=8ludffH43Tw ) it is in german.
It is verry importand that RF-Bridge-EFM8BB1 has to be flashed!!
I can read all in RFRAW ; change it in B0 an can send for examble: RFRAW AA B0 4C 04 08 136A 0280 0118.....
my sun protection works.
I hope that is a help for you ;-)

**Comment by Baggerpk** [comment]
hi.. here i am again..
sorry that my english ist not so good. because i am a german.
I found out, that it only works with older version of SONOFF RF BRIDGE .. NOT R2 !
It works with v1 (black housing) and Tasmota 9.1.0
I did it at the same way of the video ( https://www.youtube.com/watch?v=8ludffH43Tw ) it is in german.
It is verry importand that RF-Bridge-EFM8BB1 has to be flashed!!
I can read all in RFRAW ; change it in B0 an can send for examble: RFRAW AA B0 4C 04 08 136A 0280 0118.....
my sun protection works.
I hope that is a help for you ;-)

**Reply by ruun3r** [reply]
@Baggerpk:
Thank you for posting the details! I didn't had a look at it since then.
I have the new version. Maybe I can get an old one.

@sfromis: Also thank you for posting the link!
